From 58a47dbcf17e48dfeb5491def96b196494314356 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 27 Jun 2013 23:50:51 +0300
Subject: [PATCH 249/290] be: fix AST-related comment

tx's sm_group is not *necessarily* a locality sm_group (though it usually is).

Reported-by: Nikita Danilov
---
 be/tx.c | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/be/tx.c b/be/tx.c
index 4badd08..3b9cc21 100644
--- a/be/tx.c
+++ b/be/tx.c
@@ -559,7 +559,7 @@ M0_INTERNAL void tx_state_set(struct m0_be_tx *tx, enum m0_be_tx_state state)
 static void _state_set(struct m0_sm_group *grp M0_UNUSED, struct m0_sm_ast *ast)
 {
 	enum m0_be_tx_state state = *(enum m0_be_tx_state *)ast->sa_datum;
-	M0_ENTRY("state = %u", state);
+	M0_ENTRY("state=%u", state);
 	M0_PRE(IS_IN_ARRAY(state, tx_states));
 
 	/* XXX TODO: bob_of() */
@@ -571,12 +571,11 @@ M0_INTERNAL void
 m0_be__tx_state_post(struct m0_be_tx *tx, enum m0_be_tx_state state)
 {
 	/*
-	 * tx_group's fom and tx's sm may be processed by different
-	 * localities (these machines may belong different sm_groups).
-	 * See the picture:
+	 * tx_group's fom and tx's sm may belong different sm_groups (e.g.,
+	 * they may be processed by different localities).
 	 *
-	 *             locality    locality    locality
-	 *             --------    --------    --------
+	 *             locality
+	 *             --------
 	 *             sm_group    sm_group    sm_group
 	 *                | |           |         | |
 	 *                | |           |         | |
-- 
1.8.3.2

